<?xml version="1.0"?>
<launch>
    <node pkg="tl_detector" type="tl_detector.py" name="tl_detector" output="screen" cwd="node">
        <param name="model_path" value="$(find styx)../../../classifier/faster-R-CNN" />
		<param name="camera_topic" value="/image_raw" />
		<param name="isSimulation" value="False" />
    </node>
    <node pkg="tl_detector" type="light_publisher.py" name="light_publisher" output="screen" cwd="node"/>
</launch>
